bigframes/core/block_transforms.py

```diff
@@ -105,6 +106,37 @@ def indicate_duplicates(
     )
 
 
+def quantile(
+    block: blocks.Block,
+    columns: Sequence[str],
+    qs: Sequence[float],
+    grouping_column_ids: Sequence[str] = (),
+) -> blocks.Block:
+    # TODO: handle windowing and more interpolation methods
+    window = core.WindowSpec(
+        grouping_keys=tuple(grouping_column_ids),
+    )
+    quantile_cols = []
+    labels = []
+    for col in columns:
```

Comment:
Throws an exception for dataframe if column_name is larger than 30?
https://github.com/googleapis/python-bigquery-dataframes/blob/0e24036fe33c60cef08ba78e7ce69319338b6b03/bigframes/core/blocks.py#L1151

Reply:
added limit